doc/getting_started/index.rst

```diff
@@ -513,3 +525,76 @@ and are using it as the current working directory.
 
 Follow the same instructions to boot and test the images you created
 from your build.
+
+Generate the hypervisor configurations
+======================================
+
+The ACRN hypervisor leverages Kconfig to manage configurations, powered by
```

Comment:
Where do you document the configuration options and choices?

Reply:
The option documentation should go to the Kconfig files (mainly hypervisor/arch/x86/Kconfig for now). Zephyr has a script that takes a Kconfig and generates a doc in ReST.